src/storage/src/render/persist_sink.rs

```diff
@@ -115,7 +115,10 @@ pub fn render<G>(
                 .open(metadata.persist_location)
                 .await
                 .expect("could not open persist client")
-                .open_writer::<SourceData, (), Timestamp, Diff>(metadata.data_shard)
+                .open_writer::<SourceData, (), Timestamp, Diff>(
+                    metadata.data_shard,
+                    &format!("persist_sink::storage {}", src_id),
```

Comment:
nit: maybe `storage::persist_sink`, and also use `compute::persist_sink:..` for the other `persist_sink`?